script/demoScripts/demoLidoWrapper.ts

```diff
@@ -0,0 +1,228 @@
+import {
+  getContract,
+  parseUnits,
+  Narrow,
+  encodeFunctionData,
+  getAddress,
+  parseAbi,
+  formatUnits,
+  PublicClient,
+} from 'viem'
+import { randomBytes } from 'crypto'
+import dotenv from 'dotenv'
+import erc20Artifact from '../../out/ERC20/ERC20.sol/ERC20.json'
+import lidoWrapperArtifact from '../../out/LidoWrapper.sol/LidoWrapper.json'
+import { SupportedChain } from './utils/demoScriptChainConfig'
+import {
+  ensureBalance,
+  ensureAllowance,
+  executeTransaction,
+  setupEnvironment,
+  DEV_WALLET_ADDRESS,
+  getConfigElement,
+  parseAmountToHumanReadable,
+} from './utils/demoScriptHelpers'
+import diamondAbi from '../../diamond.json'
+import deploymentsOPT from '../../deployments/optimism.staging.json'
+import lidoWrapperConfig from '../../config/lidowrapper.json'
+import type { LibSwap } from '../../typechain/GenericSwapFacetV3'
+
+dotenv.config()
+
+// Successful Transactions
+// wstETH > stETH (via GenericSwapFacetV3): https://optimistic.etherscan.io/tx/0x4622d4ad989b07caae12588bab5e7e9dc8cc3cfa7eae33c3fa520a256cdbcaa2
+// stETH > wstETH (via GenericSwapFacetV3): https://optimistic.etherscan.io/tx/0xabeef0c26c8492d466bef579583a35835be03ad55a79edc8f731a6bf6e4b48d0
```

Comment:
thank you for adding successful transactions :D

Reply:
my pleasure :)